ArcGISRuntimeSDKQt_QMLSamples/UtilityNetwork/TraceUtilityNetwork/README.md

```diff
@@ -1,56 +1,66 @@
 
 ## How it works
 
 1. Create a `MapView` and connect to its `mouseClicked` signal.
-2. Create and load a `Map` that contains `FeatureLayer`(s) that are part of a utility network.
-3. Create and load a `UtilityNetwork` with the same feature service URL and map.
-4. Add a `GraphicsOverlay` with symbology that distinguishes starting points from barriers.
-5. Identify features on the map and add a `Graphic` that represents its purpose (starting point or barrier) at the location of each identified feature.
-6.  Create a `UtilityElement` for the identified feature.
-7.  Determine the type of this element using its `NetworkSource.sourceType` property.
-8.  If the element is a junction with more than one terminal, display a terminal picker. Then set the junction's `terminal` property with the selected terminal.
-9.  If an edge, set its `fractionAlongEdge` property using `GeometryEngine.fractionAlong`.
-10. Add this utility element to a collection of starting locations or barriers.
-11. Create `UtilityTraceParameters` with the selected trace type along with the collected starting locations and barriers (if applicable). 
-12. Set the `UtilityTraceParameters.traceConfiguration` with the utility tier's `traceConfiguration` property.
-13. Run a `UtilityNetwork.trace` with the specified parameters.
-14. For every `FeatureLayer` in the map, select the features using the `UtilityElement.objectId` from the filtered list of `UtilityElementTraceResult.elements`.
+2. Create and load a `ServiceGeodatabase` with a feature service URL and get tables by their layer IDs.
+3. Create a `Map` that contains `FeatureLayer`(s) created from the `ServiceGeodatabase`'s tables.
+4. Create and load a `UtilityNetwork` with the same feature service URL as the `Map`.
+5. Add a `GraphicsOverlay` with symbology that distinguishes starting locations from barriers.
+6. Identify features on the map and add a `Graphic` that represents its purpose (starting point or barrier) at the location of each identified feature.
+7.  Create a `UtilityElement` for the identified feature.
+8.  Determine the type of this element using its `NetworkSource::SourceType` property.
+9.  If the element is a junction with more than one terminal, display a terminal picker. Then set the junction's `terminal` property with the selected terminal.
+10.  If an edge, set its `fractionAlongEdge` property using `GeometryEngine::fractionAlong`.
+11. Add this `UtilityElement` to a collection of starting locations or barriers.
+12. Create `UtilityTraceParameters` with the selected trace type along with the collected starting locations and barriers (if applicable). 
+13. Set the `UtilityTraceParameters::traceConfiguration` with the utility tier's `traceConfiguration` property.
+14. Run a `UtilityNetwork::trace` with the specified parameters.
+15. For every `FeatureLayer` in the map, select the features using the `UtilityElement::objectId` from the filtered list of `UtilityElementTraceResult::elements`.
```

Comment:
One of the changes we usually make for Qml is to use the `.` notation instead of `::`. I won't comment on them all, but this is a post processing step we take:

e.g. `UtilityElement::objectId` -> `UtilityElement.objectId`